pkg/lib/dynatrace.go

```diff
@@ -192,7 +192,7 @@ func (dt *DynatraceHelper) sendDynatraceAPIRequest(apiPath string, method string
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Set("Authorization", "Api-Token "+dt.DynatraceCreds.ApiToken)
 	req.Header.Set("User-Agent", "keptn-contrib/dynatrace-service:"+os.Getenv("version"))
-	dt.Logger.Debug("Dynatrace Service version: " + os.Getenv("version"))
```

Comment:
I've removed this line on purpose, as the information about the service we are using is not relevant for every API call.